pkg/subsystem/lists/linux.go

```diff
@@ -25,14 +25,14 @@ func init() {
 //     - aspeed
 //     - coresight
 //     - imx
-//       - s32
 //     - kvmarm
 //     - mediatek
 //     - omap
+//     - perf
```

Comment:
This looks problematic. Perf becomes arm-only subsystem.

The other changes are strange (e.g. can is not net anymore), but less problematic.

Reply:
Regarding `can` : they seem to have excluded it from `netdev@` themselves:
https://elixir.bootlin.com/linux/v6.14-rc2/source/MAINTAINERS#L16544

Reply:
Fixed a silly bug in `PathMatcher`. Now the subsystem hierarchy changes are less radical.